engine/include/Engine/Core/Parsers/ParserTool.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ *	found in the top-level directory of this distribution.
+ */
+
+#pragma once
+
+#include <string>
+
+namespace GPE
+{
+inline bool isTitle(const std::string& line, std::string&& title)
+{
+    return line.substr(0, title.size()) == title;
+}
+
+template <typename T>
+T parse(const std::string& line, size_t pos = 0, size_t len = std::string::npos)
+{
+    T i;
+    std::istringstream(line.substr(pos, len)) >> i;
+    return i;
+}
+
+inline void parseName(const std::string& line, std::string& name)
+{
+    std::size_t cursorStart = line.find_first_of(" ", 0) + 1;
+    name = line.substr(cursorStart, line.size() - cursorStart); // Search first word between two space
+    if (!isalpha(name.back()))
+    {
+        name.erase(name.end() - 1);
+    }
```

Comment:
I would either rename the function to a more meaningful name, or change what is does. Currently, doing:
```C++
std::string line{"This is an average-sized string, with a few words"};
std::string sample;

parseName(line, sample);

std::cout << sample << '\n';
```

returns `is an average-sized string, with a few words`